File: src/main/java/cn/jisol/ngame/game/action/nudp/client/CNGameUDPAction.java
package cn.jisol.ngame.game.action.nudp.client;

import cn.jisol.ngame.proto.tools.AnyArrayOuterClass.*;
import cn.jisol.ngame.rpc.NGameRPCClass;
import cn.jisol.ngame.rpc.NGameRPCMethod;

@NGameRPCClass
public interface CNGameUDPAction {


    /**
     * 接收帧同步
     */
    @NGameRPCMethod
    public void nGameSyncCallBack(AnyArray nFPSInfo);

}

File: Assets/Game/Script/ncontroller/client/CNGameUDPAction.cs
using Assets.Game.plugs.NGame.tools;
using Assets.Game.Script.NGame.protobuf;
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

[NGameRPCClass]
public class CNGameUDPAction : NGameRPCIntensifier
{

    /**
     * 接收帧同步
     */
    [NGameRPCMethod]
    public void nGameSyncCallBack(AnyArray nFPSInfo) {

        Debug.Log(string.Format("CNGameUDPAction - nGameSyncCallBack {0}", nFPSInfo.Message.Count));
    
    }

}
